Dockerfile

```diff
@@ -3,6 +3,20 @@ FROM bitwalker/alpine-elixir:1.4.2
 MAINTAINER PRX <sysadmin@prx.org>
 LABEL org.prx.app="yes"
 
+# install git, aws-cli
+RUN apk update && apk --update add \
+    ca-certificates \
+    git
+
+RUN apk --no-cache update && \
+    apk --no-cache add python py-pip py-setuptools ca-certificates groff less && \
+    pip --no-cache-dir install awscli && \
```

Comment:
it is is in edge/testing - I assume if it becomes "official" it will have the versions line up.